content/_index.html

```diff
@@ -385,89 +385,92 @@ <h1>Speaker here</h1>
 		&key=AIzaSyBkIpJw0XsFQoMjgJqBoaUbD0EyHm6u4GE">
 	</iframe>
 </section>
-<!-- <section id="schedule">
+<section id="schedule">
 	<article>
 		<h1>Schedule</h1>
 		<p>Please note that this schedule is not comprehensive. Attendees please consult the booklet provided to you at the sign-in booth for the detailed schedule.</p>
 		<div class="columns">
-			<div>
+			<!-- <div>
 				<h2>Event schedule</h2>
 				<p>Coming soon!</p>
 			</div>
-			Note from the past: please include the activities so as to indicate what people are missing - even if exact times are not indicated.
+			Note from the past: please include the activities so as to indicate what people are missing - even if exact times are not indicated. -->
 			<div>
-				<h2><time datetime="2023-08-02">Wednesday 2nd</time></h2>
-				<p><time datetime="2023-08-02T15:00-07:00">3pm</time> - Check-in opens</p>
+				<h2><time datetime="2024-08-07">Wednesday 7th</time></h2>
+				<p><time datetime="2024-08-07T15:00-07:00">3pm</time> - Check-in opens</p>
+				<!-- <p><time datetime="2024-08-07T17:30-07:00">5:30pm</time> - Light Supper</p> -->
+				<p><time datetime="2024-08-07T18:30-07:00">6:30pm</time> - Icebreaker activity</p>
+				<p><time datetime="2024-08-07T19:00-07:00">7:00pm</time> - Welcome & Announcements</p>
 			</div>
-			<p title="Video will be available soon"><time datetime="2023-08-02T19:15-07:00">7:15pm</time> - Evening keynote <em></em> with Josiah Brandauer</p>
 
-			<p>A possible early morning tide-pooling adventure is in the works for either Thursday or Friday!</p>
+			<!-- <p>A possible early morning hiking adventure is in the works for either Thursday or Friday!</p> -->
 
 			<div>
-				<h2><time datetime="2023-08-03">Thursday 3rd</time></h2>
-				<p title="Video will be available soon"><time datetime="2023-08-03T09:15-07:00">9:15am</time> - Morning plenary <em></em> with Stephen Gutierrez</p>
+				<h2><time datetime="2024-08-08">Thursday 8th</time></h2>
+				<p title="Video will be available soon"><time datetime="2023-08-03T09:15-07:00">9:15am</time> - Morning plenary <em></em> with Calvin Gardiner</p>
 			</div>
-			<p><time datetime="2023-08-03T10:30-07:00">10:30am</time> - Workshops:
+			<p><time datetime="2024-08-08T10:30-07:00">10:30am</time> - Breakout Session #1
```

Comment:
Are we no longer doing workshops?

Reply:
I'm not exactly sure how it is working this year. I think it will only be one speaker leading one topic, but I'll have to wait for more info. Thus why I kept the rest of the list commented out on there.